AddAndRemoveForm/Form1.cs

```diff
@@ -17,5 +10,38 @@ public AddAndRemoveForm()
             InitializeComponent();
         }
 
+        private void buttonAdd_Click(object sender, EventArgs e)
+        {
+            string toAdd = textBoxToAddWords.Text;
```

Comment:
Empty input check?

Reply:
"+ button" is non-active if input is empty